### PR TITLE
Fix Bug For SD Encoder

### PR DESCRIPTION
**Pull request by zhen8838**

### CODE REVIEW

File: src/Native/src/ntt/cpu_runtime.cmake [hunk 1]
```diff
@@ -29,11 +29,10 @@ else()
...
 endif()
 
-add_library(nncase_cpu_runtime STATIC ${CMAKE_CURRENT_LIST_DIR}/cpu_runtime.cpp)
-target_compile_features(nncase_cpu_runtime PUBLIC cxx_std_20)
-target_include_directories(nncase_cpu_runtime PUBLIC ${CMAKE_CURRENT_LIST_DIR}/../include)
-
-if (MSVC)
-    set_property(TARGET nncase_cpu_runtime PROPERTY
-        MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
-endif()
```
Comment (sunnycase) [inline src/Native/src/ntt/cpu_runtime.cmake:39]: What's the reason of deleting these lines?

File: src/Nncase.CodeGen/Nncase.CodeGen.csproj [hunk 2]
```diff
@@ -8,7 +8,6 @@
 
   <ItemGroup>
     <PackageReference Include="Extension.Mathematics" />
-    <PackageReference Include="Razor.Templating.Core" />
```
Comment (sunnycase) [inline src/Nncase.CodeGen/Nncase.CodeGen.csproj:11]: Template engine requires this package?

File: src/Native/include/nncase/ntt/arch/aarch64/primitive_ops.h [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+// template <> struct ceil<ntt::vector<float, 8>> {
+//     ntt::vector<float, 8> operator()(const ntt::vector<float, 8>& v) const
+//     noexcept
+//     {
+//         return _mm256_ceil_ps(v);
+//     }
+// };
+
+// cos
+template <> struct cos<ntt::vector<float, 8>> {
```
Comment (sunnycase) [inline src/Native/include/nncase/ntt/arch/aarch64/primitive_ops.h:83]: Is there any performance difference between the packing of 4 and packing of 8?

File: src/Native/src/ntt/cpu_runtime.cmake (hunk 1 above)
Comment (zhen8838) [reply src/Native/src/ntt/cpu_runtime.cmake:39]: To address the multi-kernel symbol conflict problem, I removed the “nncase_cpu_runtime” and moved this source code into [Main.cpp.cshtml](https://github.com/kendryte/nncase/pull/1185/files#diff-10fa901060d9e28abca04f58bbc2fddbe42f150245d522d5bc7ae79e268329f7)

File: src/Nncase.CodeGen/Nncase.CodeGen.csproj (hunk 2 above)
Comment (zhen8838) [reply src/Nncase.CodeGen/Nncase.CodeGen.csproj:11]: no, this package will be used in Nncase.Modules.CPU

File: src/Native/include/nncase/ntt/arch/aarch64/primitive_ops.h (hunk 3 above)
Comment (zhen8838) [reply src/Native/include/nncase/ntt/arch/aarch64/primitive_ops.h:83]: I haven't tested, I just support two optional Lanes.

File: src/Native/src/ntt/cpu_runtime.cmake (hunk 1 above)
Comment (sunnycase) [reply src/Native/src/ntt/cpu_runtime.cmake:39]: Multi kernel should be linked into one elf.

File: src/Nncase.CodeGen/Nncase.CodeGen.csproj (hunk 2 above)
Comment (sunnycase) [reply src/Nncase.CodeGen/Nncase.CodeGen.csproj:11]: OK.

File: src/Native/include/nncase/ntt/arch/aarch64/primitive_ops.h (hunk 3 above)
Comment (sunnycase) [reply src/Native/include/nncase/ntt/arch/aarch64/primitive_ops.h:83]: If there is no difference, I think there is no need to repeat these codes.

Comment (zhen8838) [reply src/Native/include/nncase/ntt/arch/aarch64/primitive_ops.h]: removed redundant code

File: src/Native/src/ntt/cpu_runtime.cmake (hunk 1 above)
Comment (zhen8838) [reply src/Native/src/ntt/cpu_runtime.cmake]: restored the modification, Next PR will add the link of the module‘s kernels.

**Review by sunnycase — APPROVED** [review]

**Pull request merged**